src/pages/patterns/status-indicator-pattern/index.mdx

```diff
@@ -156,263 +226,142 @@ users, possibly hurting their overall experience.
-| Differential indicators           | Useful when users are monitoring differentials in large lists of statistics and when anything other than type would be too obtrusive.                                     | Most often used in financial dashboards for highlighting deltas or other types of data visualizations.                                                                                                                          |
-
-### Anatomy
-
-Like a button, a status indicator’s text label is the most important element, as
-it communicates progress. By default, Carbon uses sentence case for all button
-labels.
-
-Text labels, column labels (or a legend, when inline labels aren’t an option)
-are strongly recommended.
+Icon indicators consist of an icon, a shape, a meaningful color, and a
+descriptive label. They are highly effective for communication and enhance
+scannability, especially in content-heavy layouts. Common uses include
+notifications, progress indicators, data tables, task lists, and dashboard
+widgets.
 
 <Row>
 <Column colLg={8}>
 
-![Anatomy images of the five types of status indicators](./images/status_indicator_3.png)
-
-</Column>
-</Row>
-
-<Row>
-<Column colSm={2} colMd={4} colLg={4}>
-
-#### 1. Icon indicator
-
-A. Text label <br /> B. Symbol <br /> C. Shape <br /> D. Color
-
-#### 3. Badge indicator (no number)
-
-A. Shape without number <br /> B. Symbol <br /> C. Color
-
-#### 5. Differential indicator
-
-A. Text label <br /> B. Symbol\* <br /> C. Color (optional)
-
-</Column>
-<Column colSm={2} colMd={4} colLg={4}>
-
-#### 2. Badge indicator (number)
-
-A. Shape with number <br /> B. Icon <br /> C. Color
+![Carbon notification component](./images/status-icon-indicator.png)
 
-#### 4. Shape indicator
-
-A. Text label <br /> B. Shape <br /> C. Color
+<Caption>
+  Notifications are the most prevalent example of this type of status indicator.
+</Caption>
 
 </Column>
 </Row>
 
-<Caption>
-  *Differential indicators must have either a ‘+’ or ‘-’ sign, a caret, or an
-  arrow icon to indicate positive or negative values.
-</Caption>
+These severity levels are not intended to dictate the final status names in your
+product. They are suggested based on common use cases across products to help
+you define status names according to your system’s needs.
 
-## Formatting
+<StatusIndicatorTable attention="high" />
 
-### Icon indicators
+#### Labeling and type pairing
 
-Icon indicators require an icon, a shape, a meaningful color, and a descriptive
-label. Easily recognizable icons can make very effective communication tools and
-greatly improve scannability, especially with large amounts of content. Icon
-indicators are widely used in notifications, progress indicators, data tables,
-task lists and dashboard widgets.
+Icon indicators, also known as ‘contextual’ status indicators, are linked to
```

Comment:
I don't think the single quote is right here for contextual, it should be either: 
- `contextual` (code snippet)
- "contextual" (double quote)